server/testdriver/core/src/main/java/org/infinispan/server/test/core/AbstractInfinispanServerDriver.java

```diff
@@ -16,29 +31,21 @@
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
 import java.util.Arrays;
+import java.util.List;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 
-import javax.security.auth.x500.X500Principal;
-
-import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
-import org.infinispan.commons.test.CommonsTestingUtil;
-import org.infinispan.commons.test.Exceptions;
-import org.infinispan.commons.util.Util;
-import org.infinispan.lifecycle.ComponentStatus;
-import org.infinispan.security.AuthorizationPermission;
-import org.infinispan.server.Server;
-import org.infinispan.server.security.UserTool;
-import org.wildfly.security.x500.cert.BasicConstraintsExtension;
-import org.wildfly.security.x500.cert.SelfSignedX509CertificateAndSigningKey;
-import org.wildfly.security.x500.cert.X509CertificateBuilder;
-
 /**
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 10.0
  **/
 public abstract class AbstractInfinispanServerDriver implements InfinispanServerDriver {
+   public static final String DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME = "infinispan.xml";
+
+   private static final List<String> DEFAULT_INFINISPAN_CONFIG_FILES = Arrays.asList(
```

Comment:
```suggestion
   private static final List<String> DEFAULT_INFINISPAN_CONFIG_FILES = 
         Arrays.asList(DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME);
```

Reply:
I would not have a default configuration name specified. In CONTAINER mode we can simply start the server with no configuration and it will use the default `infinispan.xml`

Reply:
oki